docs/docs/text/classification.md

```diff
@@ -107,6 +103,18 @@ request = marvin.classify("Reset my password", RequestType)
 assert request == "account issue"
 ```
 
+## Returning indices
+
+In some cases, you may want to return the index of the selected label rather than the label itself:
+
+```python
+result = marvin.classify(
+    "Reset my password",
+    ["support request", "account issue", "general inquiry"],
+    return_index=True,
+)
+assert result == 1
```

Comment:
it seems like `support request` and `account issue` are both reasonable answers for this, could we make the example less ambiguous?